developer-docs-site/docs/tutorials/first-transaction-sdk.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Your First Transaction using the SDK"
+slug: "your-first-transaction-sdk"
+sidebar_position: 0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Your First Transaction
+
+This tutorial introduces the Aptos SDKs and how to generate, submit, and verify transactions submitted to the Aptos Blockchain.
+
+## Step 1: Pick an SDK
+
+* [Official Aptos Python SDK][python-sdk]
+
+## Step 2: Run the Example
+
+Each SDK provides an examples directory. This tutorial covers the `transfer-coin` example.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+      In the SDK directory run: `python -m examples.transfer-coin`
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+## Step 3: Understand the Output
+
+The following output should appear after executing the `transfer-coin` example, though some values will be different:
+
+```
+=== Addresses ===
+Alice: 0x0baec07bfc42f8018ea304ddc307a359c1c6ab20fbce598065b6cb19acff7043
+Bob: 0xc98ceafadaa32e50d06d181842406dbbf518b6586ab67cfa2b736aaddeb7c74f
+
+=== Initial Balances ===
+Alice: 20000
+Bob: 0
+
+=== Intermediate Balances ===
+Alice: 18996
+Bob: 1000
+
+=== Final Balances ===
+Alice: 17992
+Bob: 2000
+```
+
+This example demonstrates:
+
+* Initializing the REST and Faucet clients
+* The creation of two accounts: Alice and Bob
+* The funding and creation of Alice's account from a faucet
+* The creation of Bob's account from a faucet
+* The transferring of 1000 coins from Alice to Bob
+* The 4 coins for gas paid for by Alice to make that tansfer
+* Another transferring of 1000 coins from Alice to Bob
+* The additional 4 coins of gas paid for by Alice to make that transfer
+
+## Step 4: The SDK in Depth
+
+The example file leverages helper functions to interact with the [REST API][rest_spec]. This section reviews each of the calls and gives insights into functionality.
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet service for creating and funding accounts.
```

Comment:
Maybe a devnet and testnet service, since it also applies to other testnets

Reply:
one day, when they're available.